cves/CVE-2014-9604.yml

```diff
@@ -148,10 +157,10 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: There were no unit tests nearby.
+  fix: false
+  fix_answer: No fate suite
```

Comment:
Make sure to check this website as well: http://coverage.ffmpeg.org/
This was a great resource given to me on my reviews